src/commands/importFunctionApp/importFunctionApp.ts

```diff
@@ -29,7 +27,8 @@ export async function importFunctionAppToApi(node?: ApiTreeItem): Promise<void>
     ext.outputChannel.show();
     ext.outputChannel.appendLine(localize("importFunctionApp", `Import Function App started...`));
 
-    const functionApp = await getPickedFunctionApp(node);
+    ext.outputChannel.appendLine(localize("importFunctionApp", "Getting Function Apps..."));
+    const functionApp = await getPickedWebApp(node, "functionapp");
```

Comment:
Use strong types example

https://www.typescriptlang.org/docs/handbook/advanced-types.html#string-literal-types

type WebAppKind = "webapp" | "functionapp"

Reply:
we can do enum to avoid hardcode of webapp types within codes which can also be used for type checking